lib/tasks/tasks_cleanup.rake

```diff
@@ -0,0 +1,21 @@
+#
+# Usage: bundle exec rake 'cleanup:tasks'   #same as 'cleanup:tasks[7]'
+# Usage: bundle exec rake 'cleanup:tasks[days]'
+#
+require 'rake'
+
+namespace :tasks do
+  desc "Cleanup tasks older than specified days"
+  task :cleanup, [:days] => [:environment] do |_t, args|
+    days = (args.days || 7).to_i
+
+    if days.zero?
+      puts "Not allowed to delete all tasks"
+      exit
+    end
+
+    tasks = Task.where("created_at < ?", Time.zone.today - days)
```

Comment:
Should we only be cleaning up tasks that are actually done? If something somehow got stuck or failed, is there any reason to keep it around longer for debugging/investigating purposes?

Reply:
@eclarizio We could leave error tasks for a month but successful tasks could be purged in a week. We don't wan't error tasks to keep piling up

Reply:
@mkanoor That sounds fine to me, I just wanted to highlight the fact that this query would delete *all* of the old tasks, and wanted to make sure that that was what we actually wanted to do. You guys know better than I do how useful (or not) it would be to keep failed tasks around for longer.

Reply:
@eclarizio @mkanoor For now we just keep tasks for 7 days. We can change it longer in the future if we need.